src/Kafka/ConsumerGroups/components/ConsumerGroupStateLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTranslation } from "react-i18next";
+import { ConsumerGroupState } from "../types";
+
+export const ConsumerGroupStateLabel = (state: ConsumerGroupState) => {
```

Comment:
This uses hooks, it must be either a hook itself or a component. It's written like a component but you are not declaring it as a component. You are not calling this as a component either.